ansible/aws_coreos_site.yml

```diff
@@ -105,6 +148,9 @@
           - protocol: http
             load_balancer_port: 80
             instance_port: 80
+          - protocol: http
+            load_balancer_port: 8080
```

Comment:
elb would've done the mapping from 80 to 8080. Same way as we do it in the coco-provisioner. http://new-coco-registry.ft.com/__health